server/base/src/main/java/org/apache/accumulo/server/cli/ClientOnRequiredTable.java

```diff
@@ -36,6 +35,10 @@ synchronized public Instance getInstance() {
     if (instance == null) {
       return cachedInstance = HdfsZooInstance.getInstance();
     }
-    return cachedInstance = new ZooKeeperInstance(getClientConfiguration());
+    try {
+      return cachedInstance = getConnector().getInstance();
+    } catch (Exception e) {
+      throw new RuntimeException(e);
```

Comment:
This is a bad pattern to follow, for a few reasons:

1. We have multi-catch blocks now, so there's no reason to do blanket catch like this,
2. This will unnecessarily wrap `RuntimeException`s with another `RuntimeException`, and
3. You should never call `new RuntimeException(...)`; instead, pick a specific runtime exception appropriate to the circumstance.

Reply:
Should be fixed now